tools/rapids-configure-sccache

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+# A utility script that configures sccache environment variables
+
+if [ "${CI:-false}" = "false" ]; then
+  # Configure sccache for read-only mode since no credentials
+  # are available in local builds.
+  export SCCACHE_S3_NO_CREDENTIALS=true
+  export PARALLEL_LEVEL=${PARALLEL_LEVEL:-$(nproc)}
+  export RAPIDS_BUILD_TYPE=${RAPIDS_BUILD_TYPE:-"pull-request"}
+else
+  export CMAKE_CUDA_COMPILER_LAUNCHER=sccache
+  export CMAKE_CXX_COMPILER_LAUNCHER=sccache
+  export CMAKE_C_COMPILER_LAUNCHER=sccache
+  export SCCACHE_BUCKET=rapids-sccache-east
+  export SCCACHE_REGION=us-east-2
+  export SCCACHE_IDLE_TIMEOUT=32768
+  export SCCACHE_S3_USE_SSL=true
```

Comment:
We want all of these set unconditionally. The only piece that's specific to local builds is specifying that we don't have write credentials AFAIK. @ajschmidt8 can you confirm?